### PR TITLE
add docs for the platform module

### PR DESCRIPTION
**Pull request by stevendborrelli**
update documentation for `platform` extension

### CODE REVIEW

File: docs_source/content/resources/platform.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+
```
Comment (BrianHicks) [inline docs_source/content/resources/platform.md]: need header for Hugo

File: docs_source/content/resources/platform.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- `LinuxDistribution` (string)
+  Value of the [LSB](https://www.freedesktop.org/software/systemd/man/os-release.html) `ID` in `/etc/os-release`.
+  Examples: "alpine", "centos", "coreos", "debian", "nixos", "ubuntu".
+
+- `LinuxLSBLike` (list of strings)
+  Value of LSB `ID_LIKE` in `/etc/os-release` to identify related distributions.
+  Examples: `Centos` distributions return `{"rhel", "fedora"}`.
+  `Ubuntu` distributions have this value set to `{"debian"}`.
+
+- `NAME` (string)
```
Comment (BrianHicks) [inline docs_source/content/resources/platform.md]: why is this `NAME` instead of `Name`?

Comment (stevendborrelli) [inline docs_source/content/resources/platform.md]: Fixed

File: docs_source/content/platform.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
+---
+title: "platform"
+slug: "platform"
+date: "2016-10-03"
+menu:
+  main:
+    parent: content
```
Comment (BrianHicks) [inline docs_source/content/platform.md]: should be a child of `converge`

File: docs_source/content/platform.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
+---
+title: "platform"
```
Comment (BrianHicks) [inline docs_source/content/platform.md]: maybe "Platform Detection" instead? Just something user-friendly.

Comment (stevendborrelli) [inline docs_source/content/platform.md]: yes, that is better. Fixed

File: docs_source/content/resources/platform.md (hunk 1 above)
Comment (stevendborrelli) [inline docs_source/content/resources/platform.md]: added

File: docs_source/content/platform.md (hunk 3 above)
Comment (stevendborrelli) [inline docs_source/content/platform.md]: fixed

**Comment by ryane** [comment]
lgtm

**Pull request merged**